### DISCUSSION TITLE
Prima volta domotica

### DISCUSSION THREAD

**Discussion by Sunrocket** [indomus/forum#768] [GENERALE - Discussioni generiche]
Salve a tutti, premesso sono nuovissimo a questo mondo, ho visto giusto un paio di video e sto iniziando ora ad approcciarmi alla materia. Fra un annetto circa mi trasferirò nella casa che dovrò iniziare a ristrutturare a breve e quindi volevo utilizzare questo "periodo" come formativo in modo da configurare la casa nuova con i dispositivi giusti. Fatta questa lunga premessa ho comprato alcuni sensori Sonoff e l'hub, poi la chiavetta Conbee 2 e un Raspberry Pi4... ho istallato HA però ho già il primo dubbio sui sensori, cioè: L'Hub Sonoff serve? Se la chiavetta conbeeII serve per interfacciarsi ai sensori allora qual'è l'utilità dell'HUB? Ho provato ad associare due sensori all'HUB e tutto funziona il problema l'ho quando provo ad associare i sensori al DeConz, me li vede ma sono irraggiungibili. C'è una guida o una serie di video veramente a prova di tonto da guardare? Grazie a chi ha avuto la pazienza di leggere e magari di rispondere 🙂
Hardware in mio possesso:

- router TIM
- HUB Sonoff
- Chiavetta Conbee II
- 4 sensori Sonoff (sensori motion e sensori porte)
- Raspberry Pi4 con HA installato

**Comment by snakuzzo** [comment]
Per la parte Zigbee ti consiglio di utilizzare un solo hub.
Il Sonoff ZB Bridge lo puoi flashare con Tasmota ed usarlo in maniera indipendente senza legarlo al server.
Puoi sfruttarlo con ZHA se sei alle prime armi ad esempio.
La Conbee2 puoi usarla invece con Zigbee2mqtt.
A te la scelta su quale dei due usare

**Reply by Sunrocket** [reply]
Però scusa... se uso la chiavetta conbee2 non ho bisogno di nessun hub giusto?

**Reply by snakuzzo** [reply]
Ti basta quella. In questo caso il tuo hub zigbee è l'accoppiata zigbee2mqtt + conbee2 ... fine
